### ISSUE TITLE
error in ?swPressure re latitude unit

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1964]
I just noticed that the docs for `swPressure()` say that latitude can be in degrees or radians.  I think I once had an extra argument that told the unit of latitude, but that is not present now, so the doc is wrong. I'll look into whether this is done in other spots, also.

**Comment by dankelley** [comment]
There are 4 instances
```
➜  oce git:(develop) git grep "Latitude in"|grep man
man/gravity.Rd:\item{latitude}{Latitude in \eqn{^\circ}{deg}N or radians north of the
man/swDepth.Rd:\item{latitude}{Latitude in \eqn{^\circ}{deg}N or radians north of the
man/swPressure.Rd:\item{latitude}{Latitude in \eqn{^\circ}{deg}N or radians north of the
man/swZ.Rd:\item{latitude}{Latitude in \eqn{^\circ}{deg}N or radians north of the
```

**Comment by dankelley** [comment]
There are 2 instances of another form
```
➜  oce git:(develop) ✗ git grep "Vector of lat" | grep man
man/coriolis.Rd:\item{latitude}{Vector of latitudes in \eqn{^\circ}{deg}N or radians north of the equator.}
man/topoInterpolate.Rd:\item{latitude}{Vector of latitudes (in the same sign convention as used in
```

**Comment by dankelley** [comment]
Fixed in "develop" commit 3ee2269db8cb540845cb3b18e61b0418b0092b46
